src/components/pages/notifications/NotificationItem.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import {
+  Container, Row, Col, Button,
+} from 'react-bootstrap';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import moment from 'moment';
+import { connect } from 'react-redux';
+import { markOneNotifAction, getNotifsAction } from '../../../actions/notificationsActions';
+
+export class NotificationItem extends Component {
+  state = {
```

Comment:
Remove empty state and if you are not changing state make this a functional component